ui/raidboss/data/04-sb/raid/o2n.js

```diff
@@ -0,0 +1,411 @@
+'use strict';
+
+// O2N - Deltascape 2.0 Normal
+[{
+  zoneRegex: {
+    en: /^Deltascape \(V2\.0\)$/,
+  },
+  timelineFile: 'o2n.txt',
+  timelineTriggers: [
+    {
+      id: 'O2N Paranormal Wave',
+      regex: /Paranormal Wave/,
+      beforeSeconds: 5,
+      infoText: function(data) {
+        if (data.activeTank == data.me) {
+          return {
+            en: 'Tank cleave on YOU',
+          };
+        }
+        return {
+          en: 'Avoid tank cleave',
+        };
+      },
+    },
+  ],
+  triggers: [
+    {
+      // Used for avoiding Paranormal Wave.
+      id: 'O2N Tank Tracking',
+      regex: Regexes.ability({ id: '24E8', source: 'Catastrophe' }),
+      regexDe: Regexes.ability({ id: '24E8', source: 'Katastroph' }),
+      regexFr: Regexes.ability({ id: '24E8', source: 'Catastrophe' }),
+      regexJa: Regexes.ability({ id: '24E8', source: 'カタストロフィー' }),
+      regexCn: Regexes.ability({ id: '24E8', source: '灾变者' }),
+      regexKo: Regexes.ability({ id: '24E8', source: '카타스트로피' }),
+      run: function(data, matches) {
+        data.activeTank = matches.target;
+      },
+    },
+    {
+      id: 'O2N Gravitational Manipulation',
+      regex: Regexes.headMarker({ id: '0071' }),
+      condition: function(data) {
+        // The active tank shouldn't participate in the stack, as the boss would turn.
```

Comment:
I think this is maybe overthinking it.  The active tank can still participate without turning the boss if folks place it close.  Or they can hear the trigger and decide not to do it.

Reply:
I suppose so. I may have been *slightly* upset from seeing other tanks doing exactly this thing with melee in the party.